src/GaussianDistributions.jl

```diff
@@ -75,14 +77,15 @@ function rand(RNG, P::Gaussian{T}, dims) where {T}
     X
 end
 
-function rand(P::Gaussian{T}, dims) where {T}
```

Comment:
We also still need this one.

Reply:
It's handled by `rand(P::Gaussian, dims) = rand(Base.GLOBAL_RNG, P, dims)`